examples/simple-example.rs

```diff
@@ -339,7 +306,7 @@ fn main() {
     // ANCHOR: test-circuit
     // The number of rows in our circuit cannot exceed 2^k. Since our example
     // circuit is very small, we can pick a very small value here.
-    let k = 3;
+    let k = 4;
```

Comment:
`cargo run --example simple-example` is currently failing for me (`InstanceTooLarge`).